### PR TITLE
Inform setuptools that this is a binary package

### PR DESCRIPTION
**Pull request by HolgerPeters**
that needs platform-tags in wheel names.

This fixes issue #1995

### CODE REVIEW

**Comment by HolgerPeters** [comment]
Any news on this PR? travis and appveyor are now passing.

I just realized that there isn't much explanation here in plain english on what this PR accomplishes.
What it does is to tell setuptools that this package has extension modules (extension as in C extension), thus setuptools then uses the proper logic for modules that contain binaries.

**Comment by HolgerPeters** [comment]
@phunterlau @sinhrks : Do you have any comment/feedback on this PR?

**Comment by HolgerPeters** [comment]
@terrytangyuan thanks for approving. What is missing to get this merged?

**Comment by terrytangyuan** [comment]
Thanks!
